### DISCUSSION TITLE
Starting point in the centre on round workpieces

### DISCUSSION THREAD

**Discussion by OllieBFG** [arkypita/LaserGRBL#1204] [Q&A] [answered]
Hi,
Does anyone know, if I can set the zero / home location to the centre of a workpiece, in order to have the engraved picture equally distributed on the rim of a plate for example? Zero is always in the bottom left corner and if I measure my workpiece only slightly incorrect, it is not central anymore. 

The attached picture is a sample (not mine) and it's also not centred. A slight mis-measurement or mis-calculation would shift the design off-centre and the workpiece is ruined. 
![test piece](https://user-images.githubusercontent.com/59370988/102020220-9ecf9300-3d6f-11eb-9c9b-a1d38094e92f.jpg)

Any help would be greatly appreciated.

Thank you,
Oliver

**Comment by BeatSlayer** [comment]
There's a centering helper already implemented, it's at the power/speed/size gcode creation dialog, it's a small bull's-eye.

**Comment by OllieBFG** [comment]
Thanks for the fast response. I haven't noticed that yet. Will look again
and see, if I can find it.

Thanks very much for that. It's really appreciated.

Oliver

**Comment by BeatSlayer** [comment, marked as answer]
From the releases docs:

![Centering Helper](https://user-images.githubusercontent.com/8782035/96352908-8de40780-10c7-11eb-8333-46fcdcb116a8.png)
